src/gsy_e/gsy_e_core/sim_results/endpoint_buffer.py

```diff
@@ -166,6 +166,19 @@ def _get_future_orders_from_timeslot(future_orders: List, time_slot: DateTime) -
                 for order in future_orders
                 if order.time_slot == time_slot]
 
+    @staticmethod
+    def _get_current_forward_orders_from_timeslot(forward_orders: List,
+                                                  market_time_slot: DateTime,
+                                                  area: "Area") -> List:
```

Comment:
Nit:
```suggestion
    def _get_current_forward_orders_from_timeslot(
        forward_orders: List,
        market_time_slot: DateTime,
        area: "Area") -> List:
```
I think we had this conversation on Slack once, and we sort of agreed that this was our favourite type of indentation. Or do I remember wrong?

Reply:
As this part of the EndpointBuffer has changed several times during the last weeks. I'm unwilling to write tests for it now. I will definitely write tests as soon as the design is settled. Thanks for reminding that.